### DISCUSSION TITLE
[Bug]: Not working on Laravel Cloud

### DISCUSSION THREAD

**Discussion by nick-potts** [spatie/laravel-pdf#210] [Q&A]
### What happened?

I know there's not likely anything that can be done yet, but thought I'd report it ahead of time.

### How to reproduce the bug

I've tried hard to get it installed. The core issue is there's no node/npm installed on cloud. I managed to get node installed via a job, but due to the fact it builds on x64 and runs on arm, there's more issues with the puppeteer install.

### Package Version

latest

### PHP Version

n/a

### Laravel Version

11

### Which operating systems does with happen with?

Linux

**Comment by hayden-ellis** [comment]
@nick-potts Did you ever figure out a solution here?

**Reply by nick-potts** [reply]
I ended up migrating to barryvdh/laravel-dompdf

**Comment by rtaylor82** [comment]
Would love to see how you fixed it?

**Comment by dotsoftwarenl** [comment]
I just encountered this issue too. All other packages seems to work, but it does not support tailwind... Has anyone found a solution?

**Comment by MarkusHarrrison** [comment]
I also have this issue. Any ideas?

**Comment by Fr34k9** [comment]
Same here, doesnt work on laravel.cloud. Migrated as well to dompdf. Would instantly go back though.

**Comment by freekmurze** [comment]
I'd accept a PR that adds installation instructions for using Laravel PDF on Laravel Cloud to the docs.

**Comment by Fr34k9** [comment]
Its not about the docs - Laravel cloud doesnt support the installation of puppeteer.
